src/rpcrawtransaction.cpp

```diff
@@ -865,6 +866,10 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
     } else if (fHaveChain) {
         throw JSONRPCError(RPC_TRANSACTION_ALREADY_IN_CHAIN, "transaction already in block chain");
     }
+    if (IsInstantSendTxValid(tx)) {
```

Comment:
1. I don't think this should be the default behavior (at least for now), I think we should either add another optional param to `sendrawtransaction` smth like 
```
...
"sendrawtransaction \"hexstring\" ( allowhighfees instantsend )\n"
...
"3. instantsend      (boolean, optional, default=false) Use InstantSend to send this transaction\n"
...
bool fInstantSend = false;
if (params.size() > 2)
    fInstantSend = params[2].get_bool();
```
plus some param description in help.
(or create new rpc `instantsendrawtransaction` like `instantsendtoaddress` we already have, but that's probably an overkill here)

2. when (1) is done `if (IsInstantSendTxValid(tx)) { ...` should rather become
```
if (fInstantSend) {
    if (!IsInstantSendTxValid(tx)) {
        throw JSONRPCError(RPC_TRANSACTION_ERROR, "Not a valid InstantSend transaction");
    }
    mapLockRequestAccepted.insert(make_pair(hashTx, tx));
    CreateTxLockCandidate(tx);
}
```
or smth like that